optimizely/helpers/enums.py

```diff
@@ -41,3 +41,13 @@ class Errors(object):
   INVALID_VARIATION_ERROR = 'Provided variation is not in datafile.'
   UNSUPPORTED_DATAFILE_VERSION = 'Provided datafile has unsupported version. ' \
                                  'Please use SDK version 1.1.0 or earlier for datafile version 1.'
+
+
+class NotificationTypes(object):
+  """ NotificationTypes for the notification_center.NotificationCenter
+      format is NOTIFICATION TYPE: list of parameters to callback.
+  """
+  ACTIVATE = "ACTIVATE:experiment, user_id,attributes, variation, event"
```

Comment:
What was the reason for changing this back to `ACTIVATE` from `DECISION`? `ACTIVATE` is more of an Optimizely term and might not make sense to developers of other anaytics products. Instead `DECISION` is a more industry-standard term.

Reply:
@thomaszurkan-optimizely I left a comment here in case you miss it